playbooks/roles/kubernetes/tasks/remove-zowe-k8.yaml

```diff
@@ -0,0 +1,17 @@
+---
+  - name: Get namespace and service account files
+    delegate_to: localhost
+    find:
+      paths: ../containers/kubernetes/common/
+      file_type: file
+      recurse: yes
+    register: k8s_ns
+  - name: Remove the Zowe namespace and service account 
+    delegate_to: localhost
+    kubernetes.core.k8s:
+      kubeconfig: "{{ kubeconfig }}" 
+      context: "{{ k8s_context }}"
+      state: absent 
+      src: "{{ item.path }}"
+      wait: true
+    with_items: "{{ k8s_ns.files }}"
```

Comment:
We only need to delete `../containers/kubernetes/common/zowe-ns.yaml`. `zowe-sa` will be deleted after namespace is deleted.

Reply:
Okay.